examples/99_in_tool_selection/ex_in_tool_selection/main.rb

```diff
@@ -0,0 +1,364 @@
+# Copyright 2021 Trimble Inc
+# Licensed under the MIT license
+
+require 'ex_in_tool_selection/inference_lock_helper'
+
+module Examples
+  module InToolSelection
+
+    class ComponentMoveTool
+
+      # Threshold in logical screen pixels for when the mouse is considered to
+      # be dragged.
+      DRAG_THRESHOLD = 10
+
+      def activate
```

Comment:
I would have expected these to be in `initialize` (apart from the call to `update_ui`).